live-site/components/welcome-section/WelcomeSection.styles.ts

```diff
@@ -16,16 +16,23 @@ const StyledWelcomeSectionContainer = styled.div`
 
 const StyledRaceContainer = styled.div`
   background-color: ${colors.TEXT_BOX};
-  width: 40em;
+  width: 45em;
   height: 28.5em;
```

Comment:
FUN FACT: After you do all the other fixes I mentioned, we can actually remove this height - then the container will automatically be the size of the content 

See the difference:

https://user-images.githubusercontent.com/59738880/213971451-19f140f4-60e9-4d17-8f7b-bbe2003fc85c.mov

Then you can add some padding like   padding: 3em 2em; seems to be a good amount to add some more space around the edges - eezy peezy!

Reply:
maybe 2em 1em for smaller desktop sizes